public/app/features/explore/ExplorePage.tsx

```diff
@@ -55,6 +56,8 @@ export default function ExplorePage(props: GrafanaRouteComponentProps<{}, Explor
     keybindings.setupTimeRangeBindings(false);
   }, [keybindings]);
 
+  useKeyboardShortcuts();
```

Comment:
could we move 

```ts
  useEffect(() => {
    keybindings.setupTimeRangeBindings(false);
  }, [keybindings]);
```
inside `useKeyboardShortcuts` to make all the logic relative to timeranges and keybinings self-contained?